sfa_dash/blueprints/reports.py

```diff
@@ -158,24 +158,31 @@ def set_template_args(self):
             'report_template': report_template,
             'dash_url': request.url_root.rstrip('/'),
             'include_metrics_toc': False,
-            'includes_bokeh': True,
             'metadata': {
                 'report_parameters': self.metadata['report_parameters']},
         })
-        if not include_timeseries:
+        # If we're not including timeseries, or values could not be read,
+        # pop all of the timeseries/scatter spec template arguments
+        if not include_timeseries or not self.metadata['values']:
+            report_status = self.metadata.get('status')
             # display a message about omitting timeseries
             download_link = url_for('data_dashboard.download_report_html',
                                     uuid=self.metadata['report_id'])
-            script = ''
-            div = f"""<div class="alert alert-warning">
-    <strong>Warning</strong> To improve performance timeseries plots have been
-    omitted from this report. You may download a copy of this report with the
-    timeseries plots included:
-    <a href="{download_link}">Download HTML Report.</a></div>"""
-            report_kwargs.update({
-                'timeseries_div': div,
-                'timeseries_script': script,
-            })
+            report_kwargs.pop('timeseries_spec', None)
+            report_kwargs.pop('timeseries_prob_spec', None)
+            report_kwargs.pop('scatter_spec', None)
```

Comment:
This introduces a coupling of the core and dashboard design, so we should add a comment to the relevant section of core. Something like "# if adding new plots, update sfa_dash/blueprints/reports.py to exclude when data does not exist" here: https://github.com/SolarArbiter/solarforecastarbiter-core/blob/7a6feca43c9eb6c90e72de892a5384575aa90c64/solarforecastarbiter/reports/template.py#L85-L97

Reply:
Actually, looking into this I've realized we can remove this coupling. Since we're not actually interacting with the template anymore in the dashboard, and core checks if each "spec" is defined before including the plot or explanatory text, we can just adjust the core code to not define those values instead of the current setting them to `{}` and we won't have to intervene here.